responsibleai/responsibleai/_internal/_served_model_wrapper.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from raiutils.data_processing import serialize_json_safe
+
+import json
+import requests
+
+
+class ServedModelWrapper:

```

Comment:
I feel like this would more naturally live in ml-wrappers instead of the responsibleai package?

Reply:
perhaps this is similar to the existing EndpointWrapperModel:

https://github.com/microsoft/ml-wrappers/blob/main/python/ml_wrappers/model/endpoint_wrapper.py#L38

which calls a model deployed as an AzureML endpoint?

Reply:
Hmm, a few key differences, but for the moment I'd really like to retain the flexibility of making quick changes if needed. As we generalize for all tabular tasks I think it makes a ton of sense to see if we can move it to ml-wrappers (or add to an existing one).

My case is a lot simpler since it's just localhost with http, and I don't want predict or predict_proba